python_ta/cfg/visitor.py

```diff
@@ -163,20 +167,26 @@ def visit_for(self, node: astroid.For) -> None:
         self._current_block = after_for_block
 
     def visit_break(self, node: astroid.Break) -> None:
-        self._visit_continue_or_break(node)
+        self._visit_jump(node)
+
+    def visit_continue(self, node:astroid.Continue) -> None:
+        self._visit_jump(node)
 
-    def visit_continue(self, node: astroid.Continue) -> None:
-        self._visit_continue_or_break(node)
+    def visit_return(self, node: astroid.Return) -> None:
+        self._visit_jump(node)
 
-    def _visit_continue_or_break(self, node: Union[astroid.Break, astroid.Continue]) -> None:
+    def _visit_jump(self, node: Union[astroid.Break, astroid.Continue, astroid.Return]) -> None:
         old_curr = self._current_block
         for boundary, exits in reversed(self._control_boundaries):
-            if isinstance(boundary, (astroid.While, astroid.For)):
+            if type(node).__name__ not in exits:
+                continue
+            if isinstance(boundary, (astroid.For, astroid.While, astroid.FunctionDef)):
```

Comment:
You can remove this check (it should be implied by the previous check being false. This means that you can combine this part and the previous check together.